basic/knowbe4_training/impl.py

```diff
@@ -0,0 +1,27 @@
+from sym.sdk.annotations import hook, reducer
+from sym.sdk.exceptions import SymException
+from sym.sdk.integrations import knowbe4, slack
+from sym.sdk.templates import ApprovalTemplate
+
+
+# Reducers fill in the blanks that your workflow needs in order to run.
+@reducer
+def get_approvers(event):
+    """Route Sym requests to a channel specified in the sym_flow."""
+
+    # allow_self lets the requester approve themself, which is great for testing!
+    return slack.channel("#sym-requests", allow_self=True)
+
+@hook
+def on_request(event):
+    try:
+        if knowbe4.get_training_enrollments_for_user(event.user, store_purchase_id=209465, campaign_id=100345)[0]["status"] == "Passed":
```

Comment:
It would helpful to include a comment about where these IDs come from and/or what they represent when used together in this way.